services/service.js

```diff
@@ -8,30 +8,27 @@ mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern',
 
 class service {
```

Comment:
node naming scheme recommends class names to be UpperCamelCase

Reply:
fixed